modules/json/pom.xml

```diff
@@ -33,6 +33,18 @@
       <artifactId>spring-data-commons</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-parameter-names</artifactId>
```

Comment:
Can you give any rationale why we need this dependency?
I do not understand how this is related and have never used this one before.

Reply:
Done:
https://github.com/vapadwal/devon4j/blob/issue_javatime8/modules/json/pom.xml#L38